ycmd/completers/python/jedi_completer.py

```diff
@@ -56,12 +56,22 @@ def _GetJediScript( self, request_data ):
       return jedi.Script( contents, line, column, filename )
 
 
+  def _GetExtraData( self, completion ):
+      extra_data = {}
+      if completion.module_path:
+        extra_data[ 'path' ] = ToUtf8IfNeeded( completion.module_path )
+      if completion.line:
+        extra_data[ 'line' ] = completion.line
```

Comment:
It just occurred to me that this is similar to the way diagnostic locations are reported in the API; [example](https://github.com/Valloric/ycmd/blob/4cbb1374af1eedfb3600e80828d4a6c41a9690f9/ycmd/tests/diagnostics_test.py#L67). We should probably emulate that structure. If Jedi provides a column number, we should return that as well. If not, leave the `column_num` field out; we can add it in the future when this data becomes available.

So something like this:

```
location: {
  line_num: 1,
  column_num: 1,  // this can be missing
  filepath: '/foo/bar.txt'  // must be full, absolute path
}
```

The point is to have the REST API structure and field names consistent across the API.